coremltools/converters/tensorflow/test/test_tf_2x.py

```diff
@@ -11,6 +11,76 @@
 import pytest
 
 
+@unittest.skipUnless(HAS_TF_2, 'missing TensorFlow 2+.')
+class TestSingleOp(unittest.TestCase):
+    # In this class we test tensorflow 2.x op without using Keras API
+
+    def _test_coreml(self, model, input_dic=None, output_names=None):
+
+        # Get concrete function
+        concrete_func = model.__call__.get_concrete_function()
+
+        # Get function input
+        inputs = []
+        if input_dic == None:
+            for input in concrete_func.inputs:
+                name = input.name.split(':')[0]
+                shape = input.shape
+                if shape == None or any([x is None for x in input.shape.as_list()]):
+                    raise ValueError("Please specify 'input_dic' for dynamic shape input.")
+                shape = input.shape.as_list()
+                inputs.append((name, np.random.rand(*shape).astype(np.float32), shape))
+        else:
+            if not isinstance(input_dic, list):
+                raise TypeError("'input_dic' should be [(str, tensor)] type.")
+            inputs = input_dic
+
+        # Get output names
+        if output_names == None:
+            output_names = [output.name.split(':')[0] for output in concrete_func.outputs]
+        else:
+            if not isinstance(output_names, list):
+                raise TypeError("'output_names' should be [str] type.")
+        # Tensorflow predict
+        tf_inputs = [tf.convert_to_tensor(value) for name, value, shape in inputs]
+        tf_outputs = model(*tf_inputs)
+
+        # Coreml model predict
+        coreml_inputs = {name: shape for name, value, shape in inputs}
+        coreml_model = coremltools.converters.tensorflow.convert(
+            [concrete_func],
+            inputs=coreml_inputs,
+            outputs=output_names
+        )
+        coreml_outputs = coreml_model.predict({name: value for name, value, shape in inputs})
+
+        # Compare Tensorflow and Coreml
+        if not isinstance(tf_outputs, tuple):
+            tf_outputs = tuple([tf_outputs])
+        self.assertTrue(len(tf_outputs), len(coreml_outputs))
+        self.assertTrue(len(tf_outputs), len(output_names))
+        for i, output_name in enumerate(output_names):
+            np.testing.assert_almost_equal(tf_outputs[i].numpy(), coreml_outputs[output_name], decimal=3)
+
+    def test_single_output_example(self):
+
+        class model(tf.Module):
```

Comment:
Does this work for a single op model? something like this:

```
@tf.function()
def model(x):
    return tf.nn.relu(x=x)
```

Reply:
how about control flow?

Reply:
wait a minute

Reply:
Done